taxcalc/params.json

```diff
@@ -930,6 +995,32 @@
         "col_label": "",
         "value": [0]
     },
+     "_ID_Mortgage_HC":{
+        "long_name": "Mortgage interest deduction haircut",
+	"description": "This percentage can be applied to limit the amount of mortgage interest deduction allowed.",
+        "irs_ref": "",
+        "notes": "This parameter is currently used to eliminate this part of itemized deduction. ",
+        "start_year": 2013,
+        "col_var": "",
+        "row_var": "",
+        "row_label": ["2013"],
+        "cpi_inflated": false,
+        "col_label": "",
+        "value":     [0.0] 
+    },
+    "_ID_BenefitSurtax_trt":{
+        "long_name": "Deduction tax benefit surtax rate, ceiling as a percent of AGI",
```

Comment:
Let's have one 
_BenefitSurtax_crt:{
"long_name": Credit on itemized deduction benefit surtax (percent of AGI)
"description": The surtax on specified itemized deductions applies to benefits in excess of this percent of AGI. 

and one 
_BenefitSurtax_trt:{
"long_name": Surtax rate on the benefits from specified itemized deductions
"description": The benefit from specified itemized deductions exceeding the credit is taxed at this rate. A surtax rate of 1 strictly limits the benefit from specified itemized deductions to the specified credit.